### ISSUE TITLE
Cannot run abcd example from documantation

### ISSUE THREAD

**Issue by ags1234** [vlang/v#18538]
### Describe the issue

Reading documentation, the command ```v new abcd web``` creates a project that cannot be run because of an error: ```builder error: tcc: error: file 'C:\tmp\v\thirdparty\sqlite\sqlite3.c' not found```.

Indeed, the directory **sqlite** does not exist in dir **thirdparty**.

### Links

https://github.com/vlang/v/blob/master/doc/docs.md#getting-started

**Comment by dmitryuck** [comment]
check this one https://github.com/vlang/v/issues/18127

**Comment by ags1234** [comment]
@dmitryuck In my opinion it is not ok for the V documentation to instruct the user to use commands that will not work unless certain dependencies are installed. Dependencies the same documentation says nothing in advance.

The first word in the description of V is `simple`. However, it is not that simple when the second example from documentation does not work :)

Either exclude web project from doc, or include all the files needed.

**Comment by JalonSolov** [comment]
Windows is... Windows.  It will always be harder to do real, useful things on Windows than it is on Linux or Mac.  Mainly because they come with most if not all these extras already included.

That said, the docs could be improved a bit with a section stating "If you're on Windows, you need to install these dependencies first."

**Comment by spytheman** [comment]
I agree, we should improve the experience on windows. SQLite for example could be installed by running `./.github/workflows/windows-install-sqlite.bat` and that could be done by V, if it detects that it is missing.
